modules/nodes-pods-autoscaling-custom-prometheus-config.adoc

```diff
@@ -0,0 +1,199 @@
+// Module included in the following assemblies:
+//
+// * nodes/nodes-pods-autoscaling-custom.adoc
+
+:_content-type: PROCEDURE
+[id="nodes-pods-autoscaling-custom-prometheus-config_{context}"]
+= Configuring the custom metrics autoscaler to use Prometheus Cluster Monitoring
```

Comment:
This should be IMHO:
```suggestion
= Configuring the custom metrics autoscaler to use OpenShift Container Platform monitoring
```
-> http://file.rdu.redhat.com/mburke/OSDOCS-3048-custom-autoscaling-keda/monitoring/monitoring-overview.html